src/guardian-hosted/web/index.html

```diff
@@ -0,0 +1,57 @@
+
+<div class="creative creative--hosted l-side-margins with-[%numberOfElements%]-elements"
```

Comment:
I think you can try replacing `l-side-margins` with `gs-container`. Let's see what it gives.